src/utils/api.js

```diff
@@ -0,0 +1,5 @@
+const baseUrl = 'https://jy-news.onrender.com/api';
+
+export const getAllArticles = () => {
+  return fetch(`${baseUrl}/articles`).then((response) => response.json());
```

Comment:
Please use axios here so you demonstrate that you are able to use that library. It will make things simpler later as well.